### PR TITLE
Add more info to 5.0 breaking changes

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: docs/static/breaking-changes.asciidoc [hunk 1]
```diff
@@ -15,7 +17,9 @@ after installing it via a package (RPM, DEB).
 
 * **Release Packages:** When Logstash is installed via DEB, RPM packages, it uses `/usr/share/logstash` and `/var/lib/logstash` to install binaries.
 Previously it used to install in `/opt/logstash` directory. This change was done to make the user experience
-consistent with other Elastic products. Full directory layout is described in <<dir-layout>>.
+consistent with other Elastic products. Full directory layout is described in <<dir-layout>>. The source of release packages 
+have changed from `packages.elastic.co` to `artifacts.elastic.co`. For example, 5.x and all the patch releases in this series
```
Comment (dedemorton) [inline docs/static/breaking-changes.asciidoc]: Should be "The source of release packages has changed...."

File: docs/static/breaking-changes.asciidoc [hunk 2]
```diff
@@ -15,7 +17,9 @@ after installing it via a package (RPM, DEB).
 
 * **Release Packages:** When Logstash is installed via DEB, RPM packages, it uses `/usr/share/logstash` and `/var/lib/logstash` to install binaries.
 Previously it used to install in `/opt/logstash` directory. This change was done to make the user experience
-consistent with other Elastic products. Full directory layout is described in <<dir-layout>>.
+consistent with other Elastic products. Full directory layout is described in <<dir-layout>>. The source of release packages 
+have changed from `packages.elastic.co` to `artifacts.elastic.co`. For example, 5.x and all the patch releases in this series 
+source will be `https://artifacts.elastic.co/packages/5.x/apt`
```
Comment (dedemorton) [inline docs/static/breaking-changes.asciidoc]: "series source" doesn't sound right to me. Would it be correct to say, "For example, 5.x and all the patch releases in this series will available at....

**Comment by dedemorton** [comment]
Other than a couple of minor comments, LGMT.

**Comment by suyograo** [comment]
@dedemorton updated. will merge.

**Pull request closed without merging**